languages/julia/julia-tutorial/Julia-Tuor.md

```diff
@@ -0,0 +1,1184 @@
+﻿# A Julia Tour
```

Comment:
There is a typo in the filename.

Reply:
So there is.  Must have been reading Tolkien at the time...

Fixed.